data_prep/roberta_graph_dataset.py

```diff
@@ -64,23 +64,14 @@ def get_doc_embedder(self):
             encoder (RobertaModel): Roberta encoder.
         """
 
-        encoder = RobertaEncoder()
-
-        # only disables dropout
-        encoder.model.eval()
-
-        # freeze everything
-        for n, param in encoder.named_parameters():
-            param.requires_grad = False
-
         if self.roberta_checkpoint is None:
-            # use only pretrained roberta for the document embeddings
-            return encoder
-
-        # use pretrained + fine tuned roberta for the document embeddings
-        encoder.load_state_dict(self.get_encoder_state_dict())
-
-        return encoder
+            # use off-the-shelve pretrained Roberta Model
+            embedder = RobertaModel.from_pretrained('roberta-base').to(self._device)
```

Comment:
I think you can immetiately return this model here, as you are not doing any further operation on it...  then it becomes even more simple